### PR TITLE
Add machine

### PR DESCRIPTION
**Pull request by huwshimi**
Added the UI for adding a machine from the machine view.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- use the flag `/:flags:/mv/`
- open the machine view
- in the center column click "Add machine"
- enter some constraints and click Create
- a machine should be created

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3812/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3813/
Test PASSed.

File: jujugui/static/gui/src/app/components/scale-service/scale-service.js [hunk 1]
```diff
@@ -140,34 +152,8 @@ YUI.add('scale-service', function() {
...
-              onChange={this._updateState}
-              ref="ramConstraintInput"/>
-            <label htmlFor="disk-constraint">Disk (MB)</label>
-            <input type="text"
-              className="scale-service__input scale-service--constraints__input"
-              id="disk-constraint"
-              name="disk-constraint"
-              onChange={this._updateState}
-              ref="diskConstraintInput"/>
+            <juju.components.Constraints
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/scale-service/scale-service.js]: nice :)

**Comment by hatched** [comment]
:+1: QA :ok: :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**